### ISSUE TITLE
Dropdowns in modules - z-index

### ISSUE THREAD

**Issue by chmst** [joomla/cassiopeia#218]
### Steps to reproduce the issue
Make a language switcher module with dropdown.

### Expected result
The dropdown is visible.

### Actual result

![grafik](https://user-images.githubusercontent.com/1035262/97688342-4c296880-1a9a-11eb-92d2-736df1ccf365.png)

**Comment by drmenzelit** [comment]
The problem appears only when the language module has the style "card" and that is due to 
`.card {
overflow: hidden;
}`
that we added with PR #195

**Comment by infograf768** [comment]
See https://github.com/joomla/cassiopeia/pull/223#issuecomment-721039440

We should try to get something like
<img width="358" alt="Screen Shot 2020-11-04 at 12 48 36" src="https://user-images.githubusercontent.com/869724/98108372-2508d780-1e9c-11eb-9ae8-6e6590c3fc3e.png">


which implies tweaking the languages module css
for example some stuff like
```
div.mod-languages ul {
	margin: 0;
	padding: 0;
	list-style:none;
	border: solid 1px #c9c9c9;
  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
  padding-bottom: 0.5rem;
}

div.mod-languages li {
	margin-left: 0.5rem;
	margin-right: 0.5rem;
}

[dir=rtl] div.mod-languages li {
  margin-right: 0.5rem;
	text-align: right;
}

div.mod-languages ul.lang-inline li {
	display:inline-block;
}

div.mod-languages ul.lang-block li {
	display:block;
}

div.mod-languages img {
	border:none;
	display:inline-block;
}

div.mod-languages a:not([class]) {
	text-decoration: none;
}

[dir=rtl] div.mod-languages .dropdown-menu::after {
  right: .9rem;
  left: auto;
}

div.mod-languages .mod-languages__select {
width: 100%;
}
```
Also see concerning the alt:
https://github.com/joomla/joomla-cms/pull/31275

**Comment by drmenzelit** [comment]
General css changes should be done in core, because mod_languages has its own css file: build/media_source/mod_languages/css/template.css

In Cassiopeia we can do something with colours.
And we need to solve the overflow:hidden problem.

**Comment by infograf768** [comment]
> General css changes should be done in core, because mod_languages has its own css file: build/media_source/mod_languages/css/template.css

The css I posted above are mod_languages own css.

**Comment by richard67** [comment]
The same problem exists when using the metismenu e.g. on position top-a, also caused by the "card" style.

**Comment by richard67** [comment]
Overflow issue when module style = cards solved with PR #231 .
